build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
     junit: 'junit:junit:4.13.2',
     truth: 'com.google.truth:truth:1.1.3',
     robolectric: 'org.robolectric:robolectric:4.4',
-    mockito: 'org.mockito:mockito-core:2.23.0',
+    mockito: 'org.mockito:mockito-core:3.0.0',
```

Comment:
enables mock-maker-inline magic.  which allows for mocking final classes, vs weakening them as open.

hopefully temporary...